src/components/application_manager/test/commands/mobile/alert_maneuver_request_test.cc

```diff
@@ -172,18 +182,141 @@ TEST_F(AlertManeuverRequestTest, Run_ProcessingResult_SUCCESS) {
                     .asInt()));
 }
 
+TEST_F(AlertManeuverRequestTest, Run_TTSCorrect_SUCCESS) {
+  (*msg_)[am::strings::msg_params][am::strings::soft_buttons] = 0;
+  (*msg_)[am::strings::msg_params][am::strings::tts_chunks] =
+      SmartObject(smart_objects::SmartType_Array);
+
+  SmartObject tts_chunk(smart_objects::SmartType_Map);
+  tts_chunk[am::strings::text] = "correct_chunk";
+
+  (*msg_)[am::strings::msg_params][am::strings::tts_chunks]
+      .asArray()
+      ->push_back(tts_chunk);
+
+  CommandPtr command(CreateCommand<AlertManeuverRequest>(msg_));
+
+  MockAppPtr app(CreateMockApp());
+  ON_CALL(mock_app_manager_, application(_)).WillByDefault(Return(app));
+
+  ON_CALL(mock_app_manager_, GetPolicyHandler())
+      .WillByDefault(
+          ReturnRef(*static_cast<policy::PolicyHandlerInterface*>(NULL)));
```

Comment:
@indlu _Thou shalt not follow the NULL pointer, for chaos and madness await thee at its end._ [The Ten Commandments for C Programmers](https://www.lysator.liu.se/c/ten-commandments.html). Please fix this.